pilot/tools/debug/pilot_cli.go

```diff
@@ -308,10 +307,12 @@ func main() {
 			}
 		}
 	}()
-
 	var resp *xdsapi.DiscoveryResponse
 	if *configType == "lds" || *configType == "cds" {
 		pod := NewPodInfo(*resources, resolveKubeConfigPath(*kubeConfig), *proxyType)
+		if pod==nil{
```

Comment:
the formatter is probably complaining that it should be `pod == nil {`.  You can run `make lint` command locally to verify your code follows the go standards.

Reply:
@myonlyzzy if you are using an editor like Visual Studio Code or IntelliJ, these formatters should be built in.  If you have any questions feel free to ask